docs/standards.md

```diff
@@ -0,0 +1,52 @@
+# Mapping Standards
+
+This document details the standards that all contributions to Parchment mappings must follow.
+
+### Parameter Names
+
+1. Use American English for spellings
+    - Example: `color`, not `colour`, `armor`, not `armour`
+2. All names should be prefixed with `p`
+    - Example: `pLevel`, `pXPos`
+3. Names should make sense without the prefix, and should be valid java identifiers NOT reserved keywords as such:
+    - Example: `pFloat` is NOT valid, as `float` is a reserved keyword.
+    - Example: `pOs` is not a correct use of the prefix, it should be `pPos`
+4. Parameter names may be named based on the types, and the context in which they are used. They should be verbose and use complete words - do not omit essential information for brevity's sake.
+    - Example: `BlockPos pAdjacentPos, BlockPos pCurrentPos`, not `BlockPos pPos1, BlockPos pPos2`
+5. Avoid single letters, or abbreviations.
+    - Example: `pNorthWest` over `pNW`, `pMatrixStack` over `pMStack`. 
+    - Exception: common abbreviations can be used: `IO` / `NBT`, etc.
+6. Use Lower Camel Case (`lowerCamelCase`)
+7. Do not use `$` or `_` in variable names
+8. All parameters should match the regex `p[A-Z][A-Za-z0-9]+`.
```

Comment:
Point 7 is also covered by Point 8 (as the regex does not include `$` and `_`).
Also, there should be a written explanation of the regex (in a subpoint), for people not fluent in regex.
> Parameters must start with the `p` prefix, followed by an uppercase letter, followed by one or more letters or numbers. Any other characters (such as `$` or `_` are not allowed).

Reply:
> Also, there should be a written explanation of the regex (in a subpoint), for people not fluent in regex.

This is *why* there are individual points explaining each note not covered by the regex (i.e. lower camel case, no special characters, p prefix.). Removing those would require one to understand this regex - hence why Point 7 exists.